README.md

```diff
@@ -108,14 +112,15 @@ Attributes for controlling how fieldConfig works in form.
 Validation rules are documented here:
 https://ant.design/components/form/#Validation-Rules
 
-Alternatively, we support `fieldsValidator` much nicer than the default validator attribute,
-as defined by `IFieldsValidator` in `interfaces.ts`
+Alternatively, we support a `fieldsValidator` attribute in the validation rule which takes (value, fieldConfig, model) 
+and returns a boolean. This is much simpler to write than the default validation function option.
```

Comment:
why is this two lines?

Reply:
I try to keep everything under 120 wide, since that's our standard "too wide" length. The markdown render turns it into a single paragraph.

Reply:
Got it!